tasks/nunjucks.js

```diff
@@ -6,65 +6,159 @@
  * Licensed under the MIT license.
  */
 
-var nunjucks = require('nunjucks');
-var path = require('path');
-var async = require('async');
+'use strict'
 
-module.exports = function(grunt) {
-    'use strict';
+const nunjucks = require('nunjucks')
+const chalk    = require('chalk')
+const path     = require('path')
 
-    grunt.registerMultiTask('nunjucks', 'Renders nunjucks` template to HTML', function() {
-        var options = this.options();
-        var completeTask = this.async();
+module.exports = function (grunt) {
 
-        if (!options.data) {
-            grunt.log.warn('Template`s data is empty. Guess you forget to specify data option');
+    grunt.registerMultiTask('nunjucks', `Renders Nunjucks' templates to HTML`, function () {
```

Comment:
Why it's ` instead of '? I mean should it be 'Renders Nunjucks` templates to HTML'?